### DISCUSSION TITLE
Not able see the icons in some phones

### DISCUSSION THREAD

**Discussion by jalakamkiranadspl** [darioielardi/flutter_speed_dial#150] [Q&A] [answered]
Hey guys. Thanks for the superb package. I have seen a issue that kind of feels rare to me, But I didn't find any solution after searching for long time. I am using the package and in most of the phones its working good. But in some phones like Oneplus and huawei its not working. 

To replicate the issue :
Code:
`floatingActionButton: SpeedDial(
                            icon: MdiIcons.plus,
                            backgroundColor: Colors.black,
                            iconTheme: IconThemeData(color: Colors.white),
                            children: [
                            SpeedDialChild(
                              label: "xxxxxxx",
                            backgroundColor: Colors.white,
                            onTap: (){
                            },
                             child: ImageIcon(
                               AssetImage("assets/images/plants.png"),
                               size: ScreenUtil().setSp(25),
                             ),),
                              SpeedDialChild(
                                label: "xxxxxxx",
                                backgroundColor: Colors.white,
                                onTap: (){
                                },
                                child: ImageIcon(
                                  AssetImage("assets/images/disease_icon.png"),
                                  size: ScreenUtil().setSp(25),
                                ),),
                              SpeedDialChild(
                                label: "xxxxxxxx",
                                backgroundColor: Colors.white,
                                onTap: (){
                                },
                                child: Icon(Icons.wifi,color: Colors.black,),),
                          ],
                            activeIcon: Icons.highlight_remove,
                          ),`

The code formatting somehow is not coming in syntax format so here is an pastebin link if someone wants to see the code in more clearer format
https://pastebin.com/H5HjGUzR

These are the screenshots from the one plus

![image](https://user-images.githubusercontent.com/73413933/111327929-cfd3a280-8693-11eb-89bd-fed5b526c92d.png)

Thanks in advance.

**Comment by prateekmedia** [comment]
> Hey guys. Thanks for the superb package. I have seen a issue that kind of feels rare to me, ... But in some phones like Oneplus and huawei its not working...

You can try to add Center above you Image widget.

**Comment by jalakamkiranadspl** [comment]
Hey @prateekmedia, Thanks for the reply. I have tried it out, and It hasn't worked.

**Comment by jalakamkiranadspl** [comment]
@prateekmedia I have tried to replace the image with a normal icon widget and it is working somehow. But the images are not

**Comment by prateekmedia** [comment]
> @prateekmedia I have tried to replace the image with a normal icon widget and it is working somehow. But the images are not

Ok, do one thing, run with debug mode on, on that phone, by this you will see the error message. Unfortunately, I can't suggest you any other thing or improve the speeddial code without much info.

**Comment by prateekmedia** [comment]
Checkout new version, it has everything fixed.

**Comment by jalakamkiranadspl** [comment]
Sure will do that. I am sorry for not replying. I was busy with getting it
done since the deadline approached and used normal icons which worked.
Maybe the issue was with images only.

**Comment by prateekmedia** [comment, marked as answer]
Even after this it doesnt work you can set buttonChildrenSize property which was introduced in v4.for future reference.
